packages/contracts-bedrock/contracts/libraries/SafeCall.sol

```diff
@@ -35,6 +35,19 @@ library SafeCall {
         return _success;
     }
 
+    /**
+     * @notice Helper function that returns `true` if there is not enough gas remaining to cover
+     *         the minimum gas requirement for a call + an optional reserved gas amount.
+     * @param _minGas      The minimum amount of gas that may be passed to the target context.
+     * @param _reservedGas Optional amount of gas to reserve for the caller after the execution
+     *                     of the target context.
+     */
+    function checkMinGas(uint256 _minGas, uint256 _reservedGas) internal view returns (bool _halt) {
```

Comment:
Another nit: we've always avoided named vars. We should reenable that slither check.

```suggestion
    function checkMinGas(uint256 _minGas, uint256 _reservedGas) internal view returns (bool) {
```